compact.go

```diff
@@ -788,11 +784,11 @@ func (c *LeveledCompactor) populateBlock(blocks []BlockReader, meta *BlockMeta,
 				return errors.Wrap(err, "merge overlapping chunks")
 			}
 		}
-		if err := chunkw.WriteChunks(mergedChks...); err != nil {
+		if err := chunkw.WriteChunks(b[:], mergedChks...); err != nil {
```

Comment:
You mean b[:0] I suspect.

Reply:
I meant `[:]` as I was using `len` inside the function. Will change it to `[:0]` and use `cap` instead.

Reply:
Don't use cap, rather use append() to add in the value that's always there.